### DISCUSSION TITLE
Unable to add API Secret Key

### DISCUSSION THREAD

**Discussion by bradclarkuk** [MuiseDestiny/zotero-gpt#355] [Q&A] [answered]
Hello, 

Since zotero 7 has been launched as the default version i was hoping that GPT would be easier to use with it. ARIA is currently not compatible so I found your cool alternative.

The plugin is installed ok, however i can find no way to get it to accept my secret key - when i try and use the /secret key or api command and paste in the key nothing happens.

Please could someone here offer some guidance on whats going wrong here?

Many thanks,

Brad

**Comment by MuiseDestiny** [comment, marked as answer]
New Plugin: https://gitee.com/MuiseDestiny/plugins/raw/master/zotero-gpt.xpi

![image](https://github.com/user-attachments/assets/8565da14-b7d6-4078-8f00-471db8e0bade)
